clients/spark/core/src/test/scala/io/treeverse/clients/GarbageCollectorSpec.scala

```diff
@@ -41,6 +43,11 @@ class ARangeGetter(
     }
   }
 
+  def getRangeIDs(metaRangeURL: String): Iterator[String] = {
```

Comment:
Suggestion: getRangeIDsByURL - not a fan of overloading

Reply:
I understand the argument against overloading, but I think in this case it does make sense: the method signature explains what it does.

getRangeIDs [by] (**metaRangeURL** : String)
getRangeIDs [by] (**commitID**: String, **repo**: String)